src/ReactTestWrapper.js

```diff
@@ -29,7 +29,8 @@ export default class ReactTestWrapper extends TestWrapper {
   }
 
   attr (name) {
-    return this.el.getAttribute(name) || undefined
+    const value = this.el.getAttribute(name)
+    return value !== null ? value : undefined
```

Comment:
@tleunen `<a b />` `<a b={null} />` `<a b={undefined} />` `<a />` `<a b={''} />` `<a b={false} />`. a JSX value can be of any type JS supports, although the resulting HTML attributes of course will only be strings.